CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixes
+
+- [#2304: Fix acceptance test network errors](https://github.com/alphagov/govuk-prototype-kit/pull/2304)
```

Comment:
I don't think this needs a changelog entry as it's just internal.

Reply:
ok, I'll remove it

Reply:
Done